src/renderer/pages/project/components/script_editor.vue

```
@@ -54,25 +54,24 @@
-                  v-if="(mulmoValue?.beats ?? []).length !== index + 1"
-                  @click="() => positionUp(index + 1)"
-                  class="w-5 h-5 text-gray-500 hover:text-blue-500 cursor-pointer transition"
-                />
-                <Trash
-                  @click="deleteBeat(index)"
-                  class="w-5 h-5 text-gray-500 hover:text-red-500 cursor-pointer transition"
-                />
-              </div>
             </Card>
-            <div class="px-4 pt-4">
+            <div class="absolute -top-5 right-0 z-10 flex items-center gap-3 px-2 py-1 rounded border border-gray-300 bg-white shadow-sm">
```

ESLint

[error] Replace `·class="absolute·-top-5·right-0·z-10·flex·items-center·gap-3·px-2·py-1·rounded·border·border-gray-300·bg-white·shadow-sm"` with `⏎··············class="absolute·-top-5·right-0·z-10·flex·items-center·gap-3·px-2·py-1·rounded·border·border-gray-300·bg-white·shadow-sm"⏎············`

(prettier/prettier)